UnoCheck/RuntimePlatform/OperationalSystem/WindowsRuntime.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DotNetCheck.RuntimePlatform.Core;
+using DotNetCheck.RuntimePlatform.Model;
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DotNetCheck.RuntimePlatform.OperationalSystem
+{
+	public class WindowsRuntime : RuntimePlatformBase
```

Comment:
I would use a different name here. "WindowsRuntime" is a Microsoft technology and is not an OS.